.github/workflows/bump-datadog-ci.yml

```diff
@@ -17,6 +17,11 @@ on:
 jobs:
   bump-datadog-ci:
     runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: SyntheticsRunTestsTask
```

Comment:
Same as in: https://github.com/DataDog/datadog-ci-azure-devops/blob/07696bdc8743064e44dd8c50c13fc9d5ddf10d7a/.github/workflows/unit-tests.yml#L12-L14